pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -215,20 +214,24 @@ func (a *ofFlowAction) SetTunnelDst(addr net.IP) FlowBuilder {
 
 // LoadARPOperation is an action to Load data to NXM_OF_ARP_OP field.
 func (a *ofFlowAction) LoadARPOperation(value uint16) FlowBuilder {
-	a.builder.ofFlow.LoadReg(NxmFieldARPOp, uint64(value), openflow13.NewNXRange(0, 15))
+	a.builder.ofFlow.LoadReg(NxmFieldARPOp, uint64(value), openflow13.NewNXRange(0, 15), false)
 	return a.builder
 }
 
 // LoadRange is an action to Load data to the target field at specified range.
 func (a *ofFlowAction) LoadRange(name string, value uint64, rng Range) FlowBuilder {
-	a.builder.ofFlow.LoadReg(name, value, rng.ToNXRange())
+	a.builder.ofFlow.LoadReg(name, value, rng.ToNXRange(), isTunMetadataField(name))
```

Comment:
why isn't libOpenflow in charge of determining whether the name corresponds to a `tunMetadataX` field? It seems like a strange interface design choice to me.

Reply:
Since client knows if it is using a tun_metadataX field,  I want to let the caller use this parameter to cut off necessary "search" in ofnet(ofnet needs to find the tlvmap bound with the provided tun_metadataX from the tlvmaptablestatus, and reset the field length). 
Your comment makes sense to me, the interface looks strange. I will move the check down to ofnet.